src/services/sms.service.js

```diff
@@ -0,0 +1,31 @@
+import config from '../config';
+import logger from '../utils/winston';
+
+const accountSid = config.TWILIO_ACCOUNT_SID;
+const authToken = config.TWILIO_AUTH_TOKEN;
+const client = require('twilio')(accountSid, authToken);
+
+
+export const lineManagerMessenger = async (to, name) => {
+  if (config.env === 'production') {
+    await client.messages
+      .create({
+        body: `New request initiated by ${name}. Kindly review`,
+        from: config.TWILIO_PHONENUMBER,
+        to
+      })
+      .then(message => logger.info(message.sid));
```

Comment:
I'd like to suggest, to make this section a reusable function, as it can be needed in other features using SMS like Two-factor authentication, to get rid of duplications.